pkg/metrics/metrics.go

```diff
@@ -565,6 +568,80 @@ var (
 	// APILimiterProcessedRequests is the counter of the number of
 	// processed (successful and failed) requests
 	APILimiterProcessedRequests = NoOpCounterVec
+
+	// WorkQueueDepth is the depth of the workqueue
+	//
+	// We set actual metrics here instead of NoOp for the workqueue metrics
+	// because these metrics will be registered with workqueue.SetProvider
+	// by init function in watcher.go. Otherwise, we will register NoOps.
+	//
+	WorkQueueDepth = metric.NewGaugeVec(metric.GaugeOpts{
+		ConfigName: Namespace + "_" + SubsystemWorkQueue + "_depth",
+		Namespace:  Namespace,
+		Subsystem:  SubsystemWorkQueue,
+		Name:       "depth",
+		Help:       "Current depth of workqueue",
```

Comment:
nit: Consistency. Some help msgs contain punctuation others don't

Reply:
I have added punctuation to all the help messages.